pkg/proto/datadog/remoteagent/remoteagent.proto

```diff
@@ -84,3 +86,8 @@ message GetTelemetryResponse {
     string prom_text = 1;
   }
 }
+
+message PushedConfig {
```

Comment:
PushedConfig does not feel good (at least for me :)). What about ConfigUpdate?

Reply:
Updated it to `ConfigUpdate`